_posts/2024-04-27-fcmps.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Full-Chain Membership Proofs Development
+summary: A CCS to develop Full-Chain Membership Proofs has been successfully funded.
+tags: [crypto, community]
+author: Luke "Kayaba" Parker
+---
+
+Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=9&pp=iAQB), [have difficulties upon chain reorganzations](https://www.youtube.com/watch?v=6CVcirD90pg&list=PLsSYUeVwrHBnAUre2G_LYDsdo-tD0ov-y&index=4&pp=iAQB), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
```

Comment:
```suggestion
Full-Chain Membership Proofs, as a concept, is a replacement for rings within the Monero protocol. While rings have offered sender privacy to Monero since it launched, they're vulnerable to attacks such as the [EAE attack](https://www.youtube.com/watch?v=iABIcsDJKyM), [have difficulties upon chain reorganzations](https://www.youtube.com/watch?v=6CVcirD90pg), and in general enable statistical analysis (mitigated by distribution of the decoy selection algorithm). Full-Chain Membership Proofs prove the output spent is one of _any_ output on the chain, effectively removing all of these risks. This means every input goes from an immediate anonymity set of 16 to 100,000,000.
```

are we intentionally linking to the playlist instead of the video?

Reply:
Sorry for not replying earlier (though this was after merge, so no real need to comment, it'd need to be a new PR). I don't believe linking to the playlist is an issue and personally would argue it preferable for the context of and further education within Breaking Monero.

Reply:
Yeah, i didnt see the PR until after merge

I agree with that sentiment, but (dont shoot) non canonical players like newpipe open the playlist, instead of the specific video on the playlist. 

as such (being a non-canon player), i didn't request changes. Only wondering whether intentional. 

thanks

Reply:
Oh. Sorry, and thanks for the context. That may change my future preference on formatting...